src/pages/Experience/arrays/ExperienceDescriptions.js

```diff
@@ -3,27 +3,27 @@ const descriptions = [
     id: 1,
     name: "work-experience",
     description:
-      "I started my professional career in the Pharmaceutical industry, more specifically in the Clinical Trials sector." +
-      "I had the great privilege to work at awarded companies and references in this field, such as CTI, Tecnimede and Pharma Nord." +
-      "I was involved in the project management during the development of new drugs.",
+      "I started my professional career in the Pharmaceutical industry, more specifically in the Clinical Trials sector. " +
+      " I had the great privilege to work at awarded companies and references in this field and " +
```

Comment:
please add all white space only at the end of the sentence

joining line 6 with 7 would result in a double space after the dot for example

is that really what you want?

please review all other strings in this pr